pallets/gear/src/mock.rs

```diff
@@ -207,6 +209,32 @@ pub fn new_test_ext() -> sp_io::TestExternalities {
     ext
 }
 
+pub fn get_minimal_weight() -> Weight {
+    new_test_ext()
+        .execute_with(|| Gear::on_idle(System::block_number(), BlockGasLimitOf::<Test>::get()))
+}
+
+pub fn get_task_pool_add_weight() -> (Weight, Weight) {
```

Comment:
This function contract is not obvious to me. I'd rename it and remove min weight from return value.

```suggestion
pub fn get_weight_of_adding_task() -> Weight {
```